tensorboard/webapp/webapp_data_source/tb_http_client.ts

```diff
@@ -84,7 +98,8 @@ export class TBHttpClient implements TBHttpClientInterface {
     path: string,
     // Angular's HttpClient is typed exactly this way.
     body: any | null,
-    options: PostOptions | undefined = {}
+    options: PostOptions | undefined = {},
+    serializeUnder: string | undefined = undefined
```

Comment:
The plan is to follow up with some changes to calls which actually use these parameters, right?

Reply:
Yes!